### ISSUE TITLE
poor MAP resualt

### ISSUE THREAD

**Issue by amiiiirrrr** [vikrant7/mobile-vod-bottleneck-lstm#23]
hi vikrant
how did you achieved MAP=42 after two epoch?
i trained mobilenet V2 + ssdlite, but almost after 20 epoch i got MAP=30.
i started with lr=0.002 and schedulate it with Cosine. also, i used pre-trained mobilenet v2 classification (trained on imagenet) as pre-trained basenet and i loaded it's weights on my model.

**Comment by linmoxiaomo** [comment]
How did you achieved map=30? I trained ssd_basenet, the loss  almost unchanged.

**Comment by linmoxiaomo** [comment]
I trained ssd_basenet using my dataset, the loss almost unchanged. Have you met this problem?

**Comment by amiiiirrrr** [comment]
What was your dataset?
